charts/renew-ecr-k8s-creds/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
```

Comment:
You have `create: true` in values.yaml, accordingly, add if condition on top of it